extensions/VisualEditor/VisualEditor.hooks.php

```diff
@@ -46,7 +46,8 @@ public static function onSetup() {
 	 */
 	public static function onBeforePageDisplay( OutputPage &$output, Skin &$skin ) {
 		// Wikia change
-		if ( self::isAvailable( $skin ) ) {
+		// SUS-936: Only load this for article related pages (where the editor can appear)
+		if ( static::isAvailable( $skin ) && $output->isArticleRelated() ) {
 			$output->addModules( array( 'ext.visualEditor.wikia.viewPageTarget.init' ) );
 		}
 		//$output->addModules( array( 'ext.visualEditor.viewPageTarget.init' ) );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)